### PR TITLE
DALA-459 Version Updates Sprint 24

### PR DESCRIPTION
**Pull request by MarcTM01**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github.
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
  - [ ] The new feature was observed "in running software"
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one E2E Test exists testing the new feature
- [x] Documentation is updated as required
- [x] The automated deployment is updated if required
- [x] There is at least one picture for each story, which was created before coding has started
- [x] The test-script `dataland-edc-server/test/devtest.sh` still works. Run it locally!
- [x] If the behaviour of the EDC has changed from an external perspective, the dummyEDC has also been adopted to those changes

### CODE REVIEW

**Comment by MarcTM01** [comment]
Skipped update of detekt to 1.21.0 due to incompatibility with kotlin 1.7.10

File: gradle.properties [hunk 1]
```diff
@@ -1,2 +1,4 @@
 org.gradle.jvmargs=-Xmx2048m
 jacocoVersion = 0.8.8
+# Version 0.46.0 and following are not yet supported by ktLint Plugin - see https://github.com/JLLeitschuh/ktlint-gradle/issues/589
+ktlintVersion = 0.45.2
```
Comment (ppuetsch) [inline gradle.properties:4]: newline missing

File: build.gradle.kts [hunk 2]
```diff
@@ -59,14 +63,14 @@ subprojects {
 }
 
 plugins {
-    id("org.springframework.boot") version "2.7.2" apply false
+    id("org.springframework.boot") version "2.7.3" apply false
     id("io.gitlab.arturbosch.detekt") version "1.20.0"
```
Comment (ppuetsch) [inline build.gradle.kts:67]: 1.21.0?

Comment (MarcTM01) [reply build.gradle.kts:67]: Siehe PR Kommentar "Skipped update of detekt to 1.21.0 due to incompatibility with kotlin 1.7.10"

Comment (ppuetsch) [reply build.gradle.kts:67]: ah - das hast du kommentiert. Jetzt gesehen.

File: gradle.properties (hunk 1 above)
Comment (MarcTM01) [reply gradle.properties]: Added newline

Comment (MarcTM01) [reply gradle.properties]: Moment: Die Newline ist irgendwie von Intellij wieder verschluckt worden

**Pull request merged**